src/setupProxy.js

```diff
@@ -1,14 +1,19 @@
 const { createProxyMiddleware } = require('http-proxy-middleware')
 
+// adapt the proxy value to your needs, e.g. when starting a regtest setup via docker-compose.
+// use the following command to get the current ip address of the joinmarket container:
+// `docker inspect jm_regtest_joinmarket | jq ".[].NetworkSettings.Networks | to_entries[] | .value.IPAddress"`
+const target = 'localhost:28183'
```

Comment:
With the change in the docker-compose file, this isn't necessary. The ports will be available on localhost and this can stay untouched.